### PR TITLE
Add gstreamer1.0-rtsp and libgstrtspserver-1.0 packages

### PR DESCRIPTION
**Pull request by mortenfyhn**
gstreamer1.0-rtsp:
https://packages.ubuntu.com/bionic/gstreamer1.0-rtsp
https://packages.debian.org/buster/gstreamer1.0-rtsp
(Can't find on fedora, ditto for all below.)

libgstrtspserver-1.0-0:
https://packages.ubuntu.com/bionic/libgstrtspserver-1.0-0
https://packages.debian.org/buster/libgstrtspserver-1.0-0

libgstrtspserver-1.0-0-dbg:
https://packages.ubuntu.com/bionic/libgstrtspserver-1.0-0-dbg
https://packages.debian.org/buster/libgstrtspserver-1.0-0-dbg

libgstrtspserver-1.0-dev:
https://packages.ubuntu.com/bionic/libgstrtspserver-1.0-dev
https://packages.debian.org/buster/libgstrtspserver-1.0-dev

libgstrtspserver-1.0-doc:
https://packages.ubuntu.com/bionic/libgstrtspserver-1.0-doc
https://packages.debian.org/buster/libgstrtspserver-1.0-doc

### CODE REVIEW

File: rosdep/base.yaml [hunk 1]
```diff
@@ -2408,6 +2411,18 @@ libgstreamer1.0-dev:
   fedora: [gstreamer1-devel]
   gentoo: ['media-libs/gstreamer:1.0']
   ubuntu: [libgstreamer1.0-dev]
+libgstrtspserver-1.0-0:
+  debian: [libgstrtspserver-1.0-0]
+  ubuntu: [libgstrtspserver-1.0-0]
+libgstrtspserver-1.0-0-dbg:
+  debian: [libgstrtspserver-1.0-0-dbg]
+  ubuntu: [libgstrtspserver-1.0-0-dbg]
```
Comment (clalancette) [inline rosdep/base.yaml]: We don't typically add the `-dbg` packages to rosdep keys; it's just not something that downstream consumers are going to depend on.  So please remove this.

File: rosdep/base.yaml [hunk 2]
```diff
@@ -2408,6 +2411,18 @@ libgstreamer1.0-dev:
...
+libgstrtspserver-1.0-dev:
+  debian: [libgstrtspserver-1.0-dev]
+  ubuntu: [libgstrtspserver-1.0-dev]
+libgstrtspserver-1.0-doc:
+  debian: [libgstrtspserver-1.0-doc]
+  ubuntu: [libgstrtspserver-1.0-doc]
```
Comment (clalancette) [inline rosdep/base.yaml]: Same goes for documentation packages; it's just not something that we'll need to build or on the buildfarm.  So please remove this.

**Review by clalancette — CHANGES_REQUESTED** [review]
Two things to fix inline.

Also, it looks like Fedora has gstreamer1-rtsp-server: https://apps.fedoraproject.org/packages/gstreamer1-rtsp-server .  So please add that key as well (though it is unclear to me whether it goes under `gstreamer1.0-rtsp` or `libgstrtspserver-1.0-0`).

File: rosdep/base.yaml (hunk 1 above)
Comment (mortenfyhn) [reply rosdep/base.yaml]: Makes sense, will fix.

File: rosdep/base.yaml (hunk 2 above)
Comment (mortenfyhn) [reply rosdep/base.yaml]: Also makes sense.

**Comment by mortenfyhn** [comment]
I think Fedora's `gstreamer1-rtsp-server-devel` is equivalent to Ubuntu's `libgstrtspserver-1.0-dev`, based on similar contents / file lists, so I'll add it under the `libgstrtspserver-1.0-dev` key.

File: rosdep/base.yaml [hunk 3]
```diff
@@ -2408,6 +2411,13 @@ libgstreamer1.0-dev:
   fedora: [gstreamer1-devel]
   gentoo: ['media-libs/gstreamer:1.0']
   ubuntu: [libgstreamer1.0-dev]
+libgstrtspserver-1.0-0:
+  debian: [libgstrtspserver-1.0-0]
+  ubuntu: [libgstrtspserver-1.0-0]
```
Comment (clalancette) [inline rosdep/base.yaml]: Add:

```
fedora: [gstreamer1-rtsp-server]
```

**Review by clalancette — CHANGES_REQUESTED** [review]
One more fix, then I think this is good.  Thanks for iterating.

File: rosdep/base.yaml (hunk 3 above)
Comment (mortenfyhn) [reply rosdep/base.yaml]: Oh yeah, that seems right.

**Review by clalancette — APPROVED** [review]
Looks good to me, thanks!

**Review by nuclearsandwich — APPROVED** [review]

**Pull request merged**